src/CoreWCF.ConfigurationManager/tests/WSHttpBindingTests.cs

```diff
@@ -61,6 +64,13 @@ public void WSHttpBinding_WithCustomSetting()
         [Fact]
         public void WSHttpBinding_WithDefaultSetting()
         {
+            //TODO:- Check if there is a better way to skip the test
+            string frameworkDescription = RuntimeInformation.FrameworkDescription;
+            if (frameworkDescription.IndexOf(".NET Framework", StringComparison.Ordinal) >= 0)
```

Comment:
Have a look at Trait's. I added one for some tests to only be run on Windows. You can add a trait for NetCore and add that to the pipeline yml file for test runs on non-NetFx platforms.